src/dlgConnectionProfiles.cpp

```diff
@@ -1530,6 +1543,24 @@ void dlgConnectionProfiles::fillout_form()
         }
     }
 
+    mudServer = QStringLiteral("Carrion Fields");
+    if (!deletedDefaultMuds.contains(mudServer)) {
+        pM = new QListWidgetItem(mudServer);
+        pM->setFont(font);
+        pM->setForeground(QColor(Qt::white));
+        profiles_tree_widget->addItem(pM);
+        if (!hasCustomIcon(mudServer)) {
+            mi = QIcon(QStringLiteral(":/icons/carrionfields.png"));
```

Comment:
Do we have to use an actual 120x30 px icon file? I thought that we had higher resolution icons for some other MUDs that we scale down - which means that, in the future, for those with UHD/4K/Retina displays, we may be able to present higher quality images in the Connect Profiles dialogue...

Reply:
Also - do we have a Discord icon for this MUD in our Guild - so that we can show their icon - once we add them to the `(QHash<QString, QVector<QString>>) discord::mKnownGames` map - on the Discord Rich Presence when the user loads the profile and clicks the Discord opt-in?

Reply:
Hey, here's our icon for Discord. For the banner icon for Mudlet, what resolution would you prefer? The person who made the 120x30 px icon can redo it larger, she just needs to know how big it ought to be.
![cf_logo](https://user-images.githubusercontent.com/59663993/80651739-13053280-8a3c-11ea-9413-e115a547c7d2.PNG)

Reply:
Higher resolution banner
![CarrionFields](https://user-images.githubusercontent.com/59663993/80720720-eb0ce200-8ac2-11ea-8ebb-6215f30be70d.png)

Reply:
![cf_logo_large](https://user-images.githubusercontent.com/59663993/80763114-6bebce00-8b03-11ea-9303-228dabce328d.png)

Bigger icon here